2014/07/27 11:21:55 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\megaapp\protected\components\UserIdentity.php:25)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute()
#1 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#2 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(94): LoginForm->validate()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\megaapp\index.php(13): CWebApplication->run()
REQUEST_URI=/megaapp/index.php/site/login
in C:\xampp\htdocs\megaapp\protected\components\UserIdentity.php (25)
in C:\xampp\htdocs\megaapp\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php (94)
2014/07/27 11:22:44 [error] [exception.CException] exception 'CException' with message 'Property "Users.username" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('username')
#1 C:\xampp\htdocs\megaapp\protected\components\UserIdentity.php(30): CActiveRecord->__get('username')
#2 C:\xampp\htdocs\megaapp\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#14 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/megaapp/index.php/site/login
HTTP_REFERER=http://localhost/megaapp/index.php/site/login
---
2014/07/27 16:07:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "yii".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('yii')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/megaapp/index.php/yii
---
2014/07/27 16:24:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "items/view/about".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('items/view/abou...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/megaapp/index.php/items/view/about
HTTP_REFERER=http://localhost/megaapp/index.php/administrator
---
2014/07/27 16:24:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "items/view/about".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('items/view/abou...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/megaapp/index.php/items/view/about
HTTP_REFERER=http://localhost/megaapp/index.php/administrator
---
2014/07/27 16:24:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "items".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('items')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/megaapp/index.php/items/
---
2014/07/27 16:25:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "items".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('items')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/megaapp/index.php/items
---
2014/07/27 16:25:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "items".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('items')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/megaapp/index.php/items
HTTP_REFERER=http://localhost/megaapp/index.php/administrator
---
2014/07/27 17:57:11 [error] [exception.CException] exception 'CException' with message 'ItemsController contains improperly nested widget tags in its view "C:\xampp\htdocs\megaapp\protected\modules\administrator\views\items\_view.php". A CActiveForm widget does not have an endWidget() call.' in C:\xampp\htdocs\yii\framework\web\CBaseController.php:101
Stack trace:
#0 C:\xampp\htdocs\yii\framework\zii\widgets\CListView.php(291): CBaseController->renderFile('C:\\xampp\\htdocs...', Array)
#1 C:\xampp\htdocs\yii\framework\zii\widgets\CBaseListView.php(167): CListView->renderItems()
#2 [internal function]: CBaseListView->renderSection(Array)
#3 C:\xampp\htdocs\yii\framework\zii\widgets\CBaseListView.php(150): preg_replace_callback('/{(\\w+)}/', Array, '{summary}\n{sort...')
#4 C:\xampp\htdocs\yii\framework\zii\widgets\CBaseListView.php(135): CBaseListView->renderContent()
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(173): CBaseListView->run()
#6 C:\xampp\htdocs\megaapp\protected\modules\administrator\views\items\index.php(20): CBaseController->widget('zii.widgets.CLi...', Array)
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#8 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('index', Array, true)
#11 C:\xampp\htdocs\megaapp\protected\modules\administrator\controllers\ItemsController.php(128): CController->render('index', Array)
#12 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): ItemsController->actionIndex()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#14 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#16 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#17 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#18 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#19 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#20 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#21 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#22 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('administrator/i...')
#23 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#24 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#25 {main}
REQUEST_URI=/megaapp/index.php/administrator/items
HTTP_REFERER=http://localhost/megaapp/index.php/administrator/items
---
2014/07/27 18:28:31 [error] [exception.CException] exception 'CException' with message 'Property "Items.title" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('title')
#1 C:\xampp\htdocs\megaapp\protected\views\site\items.php(4): CActiveRecord->__get('title')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('items', Array, true)
#6 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(114): CController->render('items', Array)
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionItems()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('items')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/items')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#14 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/megaapp/index.php/site/items
---
2014/07/27 18:28:57 [error] [exception.CException] exception 'CException' with message 'Property "Items.title" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('title')
#1 C:\xampp\htdocs\megaapp\protected\views\site\items.php(4): CActiveRecord->__get('title')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('items', Array, true)
#6 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(114): CController->render('items', Array)
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionItems()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('items')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/items')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#14 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/megaapp/index.php/site/items
---
2014/07/27 18:42:52 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "bucket".' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('bucket', NULL, true)
#1 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(124): CController->render('bucket')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionBucket()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('bucket')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/bucket')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/megaapp/index.php/site/bucket
HTTP_REFERER=http://localhost/megaapp/index.php/site/items?view=about
---
2014/07/27 18:50:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "megaapp".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('megaapp')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('megaapp')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/megaapp/in...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/megaapp/index.php/site/megaapp/index.php/site/addItem
HTTP_REFERER=http://localhost/megaapp/index.php/site/items?view=about
---
2014/07/27 18:50:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "megaapp".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('megaapp')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('megaapp')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/megaapp/in...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/megaapp/index.php/site/megaapp/index.php/site/addItem
HTTP_REFERER=http://localhost/megaapp/index.php/site/items?view=about
---
2014/07/27 18:50:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "site".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('site')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('site')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/site/addIt...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/megaapp/index.php/site/site/addItem
HTTP_REFERER=http://localhost/megaapp/index.php/site/items?view=about
---
2014/07/27 18:51:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "site".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('site')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('site')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/site/addIt...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/megaapp/index.php/site/site/addItem
HTTP_REFERER=http://localhost/megaapp/index.php/site/items?view=about
---
2014/07/27 18:51:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "site".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('site')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('site')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/site/addIt...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/megaapp/index.php/site/site/addItem
HTTP_REFERER=http://localhost/megaapp/index.php/site/items?view=about
---
2014/07/27 18:51:08 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "ssd".' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('ssd', NULL, true)
#1 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(124): CController->render('ssd')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionBucket()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('bucket')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/bucket')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/megaapp/index.php/site/bucket
HTTP_REFERER=http://localhost/megaapp/index.php/site/items?view=about
---
2014/07/27 18:53:28 [error] [php] Indirect modification of overloaded element of CHttpSession has no effect (C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php:119)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 C:\xampp\htdocs\megaapp\index.php(13): CWebApplication->run()
REQUEST_URI=/megaapp/index.php/site/addItem
in C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php (119)
in C:\xampp\htdocs\megaapp\index.php (13)
2014/07/27 18:57:10 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('addItem')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/addItem')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/megaapp/index.php/site/addItem
HTTP_REFERER=http://localhost/megaapp/index.php/site/items
---
2014/07/27 18:57:15 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('addItem')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/addItem')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/megaapp/index.php/site/addItem
HTTP_REFERER=http://localhost/megaapp/index.php/site/items
---
2014/07/27 18:57:45 [error] [exception.CException] exception 'CException' with message 'Property "SiteController.attributes" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(125): CComponent->__get('attributes')
#1 [internal function]: SiteController->actionAddItem('0')
#2 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(SiteController), Array)
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(SiteController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('addItem')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/addItem')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/megaapp/index.php/site/addItem/0
HTTP_REFERER=http://localhost/megaapp/index.php/site/items?view=about
---
2014/07/27 18:57:48 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('addItem')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/addItem/it...')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/megaapp/index.php/site/addItem/items
HTTP_REFERER=http://localhost/megaapp/index.php/site/items?view=about
---
2014/07/27 18:59:08 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('addItem')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/addItem/it...')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/megaapp/index.php/site/addItem/items
HTTP_REFERER=http://localhost/megaapp/index.php/site/items?view=about
---
2014/07/27 18:59:34 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('addItem')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/addItem/si...')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/megaapp/index.php/site/addItem/site/items
HTTP_REFERER=http://localhost/megaapp/index.php/site/items?view=about
---
2014/07/27 18:59:35 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('addItem')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/addItem/si...')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/megaapp/index.php/site/addItem/site/items
HTTP_REFERER=http://localhost/megaapp/index.php/site/items?view=about
---
2014/07/27 18:59:49 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('addItem')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/addItem/si...')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/megaapp/index.php/site/addItem/site/items
HTTP_REFERER=http://localhost/megaapp/index.php/site/items?view=about
---
2014/07/27 18:59:54 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "bucket".' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('bucket', NULL, true)
#1 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(132): CController->render('bucket')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionBucket()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('bucket')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/bucket')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/megaapp/index.php/site/bucket
HTTP_REFERER=http://localhost/megaapp/index.php/site/items
---
2014/07/27 19:08:38 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\megaapp\protected\views\site\bucket.php:3)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(135): SiteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionBucket()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\megaapp\index.php(13): CWebApplication->run()
REQUEST_URI=/megaapp/index.php/site/bucket
in C:\xampp\htdocs\megaapp\protected\views\site\bucket.php (3)
in C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php (135)
in C:\xampp\htdocs\megaapp\index.php (13)
2014/07/27 19:11:02 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\megaapp\protected\views\site\bucket.php:3)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(135): SiteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionBucket()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\megaapp\index.php(13): CWebApplication->run()
REQUEST_URI=/megaapp/index.php/site/bucket
in C:\xampp\htdocs\megaapp\protected\views\site\bucket.php (3)
in C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php (135)
in C:\xampp\htdocs\megaapp\index.php (13)
2014/07/27 19:12:08 [error] [php] Undefined variable: item (C:\xampp\htdocs\megaapp\protected\views\site\bucket.php:3)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(134): SiteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionBucket()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\megaapp\index.php(13): CWebApplication->run()
REQUEST_URI=/megaapp/index.php/site/bucket
in C:\xampp\htdocs\megaapp\protected\views\site\bucket.php (3)
in C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php (134)
in C:\xampp\htdocs\megaapp\index.php (13)
2014/07/27 19:12:09 [error] [php] Undefined variable: item (C:\xampp\htdocs\megaapp\protected\views\site\bucket.php:3)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(134): SiteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionBucket()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\megaapp\index.php(13): CWebApplication->run()
REQUEST_URI=/megaapp/index.php/site/bucket
in C:\xampp\htdocs\megaapp\protected\views\site\bucket.php (3)
in C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php (134)
in C:\xampp\htdocs\megaapp\index.php (13)
2014/07/27 19:12:34 [error] [php] Undefined variable: item (C:\xampp\htdocs\megaapp\protected\views\site\bucket.php:3)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(134): SiteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionBucket()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\megaapp\index.php(13): CWebApplication->run()
REQUEST_URI=/megaapp/index.php/site/bucket
in C:\xampp\htdocs\megaapp\protected\views\site\bucket.php (3)
in C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php (134)
in C:\xampp\htdocs\megaapp\index.php (13)
2014/07/27 20:09:41 [error] [php] Undefined variable: count (C:\xampp\htdocs\megaapp\protected\views\site\bucket.php:4)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(106): SiteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionBucket()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\megaapp\index.php(13): CWebApplication->run()
REQUEST_URI=/megaapp/index.php/site/bucket
in C:\xampp\htdocs\megaapp\protected\views\site\bucket.php (4)
in C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php (106)
in C:\xampp\htdocs\megaapp\index.php (13)
2014/07/27 20:09:52 [error] [php] Undefined variable: count (C:\xampp\htdocs\megaapp\protected\views\site\bucket.php:4)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(106): SiteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionBucket()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\megaapp\index.php(13): CWebApplication->run()
REQUEST_URI=/megaapp/index.php/site/bucket
in C:\xampp\htdocs\megaapp\protected\views\site\bucket.php (4)
in C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php (106)
in C:\xampp\htdocs\megaapp\index.php (13)
2014/07/27 20:09:56 [error] [exception.CException] exception 'CException' with message 'Property "OrderForm.PhoneNumber" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2529): CComponent->__get('PhoneNumber')
#1 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2252): CHtml::resolveValue(Object(OrderForm), 'PhoneNumber')
#2 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1434): CHtml::activeInputField('text', Object(OrderForm), 'PhoneNumber', Array)
#3 C:\xampp\htdocs\megaapp\protected\views\site\order.php(7): CHtml::activeTextField(Object(OrderForm), 'PhoneNumber')
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#7 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('order', Array, true)
#8 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(122): CController->render('order', Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionOrder()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('order')
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/order')
#15 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#16 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/megaapp/index.php/site/order/
HTTP_REFERER=http://localhost/megaapp/index.php/site/bucket
---
2014/07/27 20:10:10 [error] [exception.CException] exception 'CException' with message 'Property "OrderForm.PhoneNumber" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2529): CComponent->__get('PhoneNumber')
#1 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2252): CHtml::resolveValue(Object(OrderForm), 'PhoneNumber')
#2 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1434): CHtml::activeInputField('text', Object(OrderForm), 'PhoneNumber', Array)
#3 C:\xampp\htdocs\megaapp\protected\views\site\order.php(7): CHtml::activeTextField(Object(OrderForm), 'PhoneNumber')
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#7 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('order', Array, true)
#8 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(122): CController->render('order', Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionOrder()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('order')
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/order')
#15 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#16 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/megaapp/index.php/site/order/
HTTP_REFERER=http://localhost/megaapp/index.php/site/bucket
---
2014/07/27 20:10:11 [error] [exception.CException] exception 'CException' with message 'Property "OrderForm.PhoneNumber" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2529): CComponent->__get('PhoneNumber')
#1 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2252): CHtml::resolveValue(Object(OrderForm), 'PhoneNumber')
#2 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1434): CHtml::activeInputField('text', Object(OrderForm), 'PhoneNumber', Array)
#3 C:\xampp\htdocs\megaapp\protected\views\site\order.php(7): CHtml::activeTextField(Object(OrderForm), 'PhoneNumber')
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#7 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('order', Array, true)
#8 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(122): CController->render('order', Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionOrder()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('order')
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/order')
#15 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#16 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/megaapp/index.php/site/order/
HTTP_REFERER=http://localhost/megaapp/index.php/site/bucket
---
2014/07/27 20:10:11 [error] [exception.CException] exception 'CException' with message 'Property "OrderForm.PhoneNumber" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2529): CComponent->__get('PhoneNumber')
#1 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2252): CHtml::resolveValue(Object(OrderForm), 'PhoneNumber')
#2 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1434): CHtml::activeInputField('text', Object(OrderForm), 'PhoneNumber', Array)
#3 C:\xampp\htdocs\megaapp\protected\views\site\order.php(7): CHtml::activeTextField(Object(OrderForm), 'PhoneNumber')
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#7 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('order', Array, true)
#8 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(122): CController->render('order', Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionOrder()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('order')
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/order')
#15 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#16 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/megaapp/index.php/site/order/
HTTP_REFERER=http://localhost/megaapp/index.php/site/bucket
---
2014/07/27 20:10:11 [error] [exception.CException] exception 'CException' with message 'Property "OrderForm.PhoneNumber" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2529): CComponent->__get('PhoneNumber')
#1 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2252): CHtml::resolveValue(Object(OrderForm), 'PhoneNumber')
#2 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1434): CHtml::activeInputField('text', Object(OrderForm), 'PhoneNumber', Array)
#3 C:\xampp\htdocs\megaapp\protected\views\site\order.php(7): CHtml::activeTextField(Object(OrderForm), 'PhoneNumber')
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#7 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('order', Array, true)
#8 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(122): CController->render('order', Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionOrder()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('order')
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/order')
#15 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#16 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/megaapp/index.php/site/order/
HTTP_REFERER=http://localhost/megaapp/index.php/site/bucket
---
2014/07/27 20:10:12 [error] [exception.CException] exception 'CException' with message 'Property "OrderForm.PhoneNumber" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2529): CComponent->__get('PhoneNumber')
#1 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2252): CHtml::resolveValue(Object(OrderForm), 'PhoneNumber')
#2 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1434): CHtml::activeInputField('text', Object(OrderForm), 'PhoneNumber', Array)
#3 C:\xampp\htdocs\megaapp\protected\views\site\order.php(7): CHtml::activeTextField(Object(OrderForm), 'PhoneNumber')
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#7 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('order', Array, true)
#8 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(122): CController->render('order', Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionOrder()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('order')
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/order')
#15 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#16 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/megaapp/index.php/site/order/
---
2014/07/27 20:10:14 [error] [exception.CException] exception 'CException' with message 'Property "OrderForm.PhoneNumber" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2529): CComponent->__get('PhoneNumber')
#1 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2252): CHtml::resolveValue(Object(OrderForm), 'PhoneNumber')
#2 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1434): CHtml::activeInputField('text', Object(OrderForm), 'PhoneNumber', Array)
#3 C:\xampp\htdocs\megaapp\protected\views\site\order.php(7): CHtml::activeTextField(Object(OrderForm), 'PhoneNumber')
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#7 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('order', Array, true)
#8 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(122): CController->render('order', Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionOrder()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('order')
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/order')
#15 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#16 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/megaapp/index.php/site/order/
---
2014/07/27 20:10:23 [warning] [application] Failed to set unsafe attribute "PhoneNumber" of "OrderForm".
in C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php (115)
in C:\xampp\htdocs\megaapp\index.php (13)
2014/07/27 20:10:23 [error] [exception.CException] exception 'CException' with message 'Property "OrderForm.phone" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\megaapp\protected\models\OrderForm.php(40): CComponent->__get('phone')
#1 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): OrderForm->phoneValidation('phone', Array)
#2 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(OrderForm), 'phone')
#3 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(OrderForm), NULL)
#4 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(117): CModel->validate()
#5 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionOrder()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('order')
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/order')
#11 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/megaapp/index.php/site/order/
HTTP_REFERER=http://localhost/megaapp/index.php/site/order/
---
2014/07/27 20:11:28 [error] [php] strpos() expects parameter 1 to be string, array given (C:\xampp\htdocs\yii\framework\web\CController.php:706)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(867): SiteController->getViewFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(118): SiteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionOrder()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\megaapp\index.php(13): CWebApplication->run()
REQUEST_URI=/megaapp/index.php/site/order/
in C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php (118)
in C:\xampp\htdocs\megaapp\index.php (13)
2014/07/27 20:12:14 [error] [php] strpos() expects parameter 1 to be string, array given (C:\xampp\htdocs\yii\framework\web\CController.php:706)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(867): SiteController->getViewFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(118): SiteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionOrder()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\megaapp\index.php(13): CWebApplication->run()
REQUEST_URI=/megaapp/index.php/site/order/
in C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php (118)
in C:\xampp\htdocs\megaapp\index.php (13)
2014/07/27 20:13:16 [error] [php] strpos() expects parameter 1 to be string, array given (C:\xampp\htdocs\yii\framework\web\CController.php:706)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(867): SiteController->getViewFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(118): SiteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionOrder()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\megaapp\index.php(13): CWebApplication->run()
REQUEST_URI=/megaapp/index.php/site/order/
in C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php (118)
in C:\xampp\htdocs\megaapp\index.php (13)
2014/07/27 20:13:25 [error] [php] strpos() expects parameter 1 to be string, array given (C:\xampp\htdocs\yii\framework\web\CController.php:706)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(867): SiteController->getViewFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(118): SiteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionOrder()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\megaapp\index.php(13): CWebApplication->run()
REQUEST_URI=/megaapp/index.php/site/order/
in C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php (118)
in C:\xampp\htdocs\megaapp\index.php (13)
2014/07/27 20:13:35 [error] [php] strpos() expects parameter 1 to be string, array given (C:\xampp\htdocs\yii\framework\web\CController.php:706)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(867): SiteController->getViewFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(118): SiteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionOrder()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\megaapp\index.php(13): CWebApplication->run()
REQUEST_URI=/megaapp/index.php/site/order/
in C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php (118)
in C:\xampp\htdocs\megaapp\index.php (13)
2014/07/27 20:14:14 [error] [php] strpos() expects parameter 1 to be string, array given (C:\xampp\htdocs\yii\framework\web\CController.php:706)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(867): SiteController->getViewFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php(118): SiteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionOrder()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\xampp\htdocs\megaapp\index.php(13): CWebApplication->run()
REQUEST_URI=/megaapp/index.php/site/order/
in C:\xampp\htdocs\megaapp\protected\controllers\SiteController.php (118)
in C:\xampp\htdocs\megaapp\index.php (13)
2014/07/27 20:15:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "sucsess".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('sucsess')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('sucsess')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/sucsess')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\megaapp\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/megaapp/index.php/site/sucsess
HTTP_REFERER=http://localhost/megaapp/index.php/site/order/
---
